README.md

```diff
@@ -8,19 +8,50 @@
 
 # Installation 
 
-1. `npm install screepsmod-auth` in your server folder.
-2. Thats it!
+1. Either run `npm install screepsmod-auth` in your screeps private server folder or subscribe the Steam workshop  https://steamcommunity.com/sharedfiles/filedetails/?id=800390576 if your using the in-game private server.
```

Comment:
This should probably be split into two subsections, `npm version` and `steam version`

Reply:
Done